src/topology/locally_constant/algebra.lean

```diff
@@ -151,4 +153,27 @@ function.injective.distrib_mul_action coe_fn_add_monoid_hom coe_injective (λ _
 instance [semiring R] [add_comm_monoid Y] [module R Y] : module R (locally_constant X Y) :=
 function.injective.module R coe_fn_add_monoid_hom coe_injective (λ _ _, rfl)
 
+section algebra
+
+variables [comm_semiring R] [semiring Y] [algebra R Y]
+
+/-- The constant-function embedding, as a ring hom.  -/
+@[simps] def const_ring_hom : Y →+* locally_constant X Y :=
+{ to_fun    := const X,
+  map_one'  := rfl,
+  map_mul'  := λ _ _, rfl,
+  map_zero' := rfl,
+  map_add'  := λ _ _, rfl, }
+
+instance : algebra R (locally_constant X Y) :=
+{ to_ring_hom := const_ring_hom.comp $ algebra_map R Y,
+  commutes'   := by { intros, ext, exact algebra.commutes' _ _, },
+  smul_def'   := by { intros, ext, exact algebra.smul_def' _ _, }, }
+
+@[simp] lemma coe_algebra_map :
+  (algebra_map R _ : R → locally_constant X Y) = (const X) ∘ (algebra_map R Y) :=
+rfl
```

Comment:
The statement I'd expect this lemma to have is something like
```suggestion
@[simp] lemma coe_algebra_map (r : R) :
  ⇑(algebra_map R (locally_constant X Y) r) = algebra_map R (X → Y) r :=
rfl
```
(untested)

Reply:
OK sure. (It works btw.)